tests/unit/mathematics/linalg/operations/Eigen3_operations_unittest.cc

```diff
@@ -406,6 +406,44 @@ TEST(LinalgBackendEigen, SGMatrix_cholesky_solver)
 	EXPECT_EQ(x_ref.size(), x_cal.size());
 }
 
+TEST(LinalgBackendEigen, SGMatrix_ldlt_solver)
```

Comment:
i wonder why there is "SGMatrix" in the test name?

Reply:
Just followed other test cases like `SGMatrix_cholesky_solver`